### PR TITLE
Add schemas for Glean history and bookmarks sync pings

### PR DESCRIPTION
**Pull request by linacambridge**
These were added in mozilla-mobile/android-components#3092. I cribbed this PR from mozilla-services/mozilla-pipeline-schemas#325, please let me know if there's anything that needs to happen next for us to see the data in re:dash!

(CC @Dexterp37 @mdboom)

Checklist for reviewer:

- [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

**Comment by linacambridge** [comment]
Hmm, was there a reason underscores were disallowed in schema names? I'm wondering if that's causing the failure...but also, the [debug ping viewer](https://debug-ping-preview.firebaseapp.com/pings/a90b3646-dc2e-437c-84af-782d97e1c917/lina) reports `No such docType: org-mozilla-fenix-debug/bookmarks_sync`, is there something else I should call it if I can't use underscores?

File: CMakeLists.txt [hunk 1]
```diff
@@ -53,9 +53,9 @@ endforeach()
 file(GLOB_RECURSE SCHEMAS LIST_DIRECTORIES FALSE RELATIVE "${CMAKE_SOURCE_DIR}/templates" "${CMAKE_SOURCE_DIR}/templates/*")
 foreach(ITEM IN ITEMS ${SCHEMAS})
   if(NOT ITEM MATCHES "^(include/|.*/\\.[^/])" )
-    string(REGEX MATCH "/([-a-zA-Z0-9]+\\.[1-9][0-9]*\\.schema\\.json|[-a-zA-Z0-9_]+\\.[1-9][0-9]*\\.parquetmr\\.txt|README.*)$" MATCH ${ITEM})
+    string(REGEX MATCH "/([-a-zA-Z0-9_]+\\.[1-9][0-9]*\\.schema\\.json|[-a-zA-Z0-9_]+\\.[1-9][0-9]*\\.parquetmr\\.txt|README.*)$" MATCH ${ITEM})
```
Comment (Dexterp37) [inline CMakeLists.txt]: Mh, this is for the schema filenames, I think. @trink , thoughts?
I think, at the very least, this should allow the same characters in the ping filename that we allow for the ping type [here](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/2dd16ff73b5c857a4d9b90325e20f32145fbbbaf/schemas/glean/baseline/baseline.1.schema.json#L754-L756).

Comment (trink) [reply CMakeLists.txt]: The current regex matches what is documented here (the docs actually need to be updated to include the numeric range): http://docs-origin.telemetry.mozilla.org/cookbooks/new_ping.html?highlight=namespace#choose-a-namespace-and-doctype.  There is no restriction in the ingestion ETL the URL is simply split on the `/` and the corresponding values are used for the schema lookup.

Comment (Dexterp37) [reply CMakeLists.txt]: I see, thank you @trink ! Then I think it's now up to @fbertsch or @mreid-moz to decide if this is ok to extend or not :)

Comment (linacambridge) [reply CMakeLists.txt]: > There is no restriction in the ingestion ETL the URL is simply split on the / and the corresponding values are used for the schema lookup.

That's what I was wondering about. 😄 The [Glean debug ping viewer](https://debug-ping-preview.firebaseapp.com/pings/UNKNOWN/lina) shows errors like:

> ParsePayload com.mozilla.telemetry.schemas.SchemaNotFoundException: No such docType: org-mozilla-fenix-debug/bookmarks_sync

I guess debug builds [wouldn't send telemetry anyway](https://github.com/mozilla-mobile/fenix/blob/a58a77317e49814c3e1f046b2d4fda2a49abcff5/app/build.gradle#L207-L211), but, for release builds—does the file also need to be named `org-mozilla-fenix/bookmarks_sync`, or can it have a different name?

Comment (Dexterp37) [reply CMakeLists.txt]: It needs to have the same name as the ping name in the `pings.yaml` file, and it must be in a directory with the application id as the directory name. Your current PR seems to look good.

**Pull request merged**